modape/scripts/modis_window.py

```diff
@@ -168,7 +168,7 @@ def cli(src: str,
 
     gdal_kwargs = {}
     if gdal_kwarg:
```

Comment:
Not sure if I see the point of checking `gdal_kwarg` again, this is done in line 170. Lines 171 ff are just to separate `dict` and tuple

Reply:
Very true. I looked at it as if it were a typo: `is` should be `and`. That change solves the problem, but the first check is redundant indeed.